*: add tidb_opt_use_invisible_indexes (#16762)
pingcap/docs: system-variables.md

- Default value: `24.0`
- Indicates the concurrency number of TiFlash computation. This variable is internally used in the Cost Model, and it is NOT recommended to modify its value.

### tidb_opt_use_invisible_indexes

- Scope: SESSION
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
- Default Value: OFF

> - This variable is used to set whether the optimizer is allowed to use invisible indexes. Invisible indexes refer to indexes created by the user, but such indexes are not used by the optimizer. After enabling this option, the optimizer will attempt to use invisible indexes.

Reviewer: ```suggestion
- This variable is used to set whether the optimizer is allowed to use invisible indexes. Invisible indexes refer to indexes created by the user, but such indexes are ignored by the optimizer when accessing the base table. After enabling this option, the optimizer will attempt to use invisible indexes.
```

- Default value: `24.0`
- Indicates the concurrency number of TiFlash computation. This variable is internally used in the Cost Model, and it is NOT recommended to modify its value.

### tidb_opt_use_invisible_indexes <span class="version-mark">New in v8.0.0</span>

> - Scope: SESSION

Reviewer 2: ```suggestion
- Scope: SESSION
- Persists to cluster: 
```

- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
- Type: Boolean
- Default value: `OFF`

> - This variable controls whether whether the optimizer can select [invisible indexes](/sql-statements/sql-statement-create-index.md#invisible-index) for query optimization in the current session. Invisible indexes are maintained by DML statements, but will not be used by the query optimizer. This is useful in scenarios where you want to double-check before removing an index permanently. When the variable is set to `ON`, the optimizer can select invisible indexes for query optimization in the session.

Reviewer 2: ```suggestion
- This variable controls whether the optimizer can select [invisible indexes](/sql-statements/sql-statement-create-index.md#invisible-index) for query optimization in the current session. Invisible indexes are maintained by DML statements, but will not be used by the query optimizer. This is useful in scenarios where you want to double-check before removing an index permanently. When the variable is set to `ON`, the optimizer can select invisible indexes for query optimization in the session.
```
